src/scheduler/job_info.cpp

```diff
@@ -3116,7 +3057,7 @@ find_and_preempt_jobs(status *policy, int pbs_sd, resource_resv *hjob, server_in
 			}
 		}
 
-		if ((preempt_jobs_reply = pbs_preempt_jobs(pbs_sd, preempt_jobs_list)) == NULL) {
+		if ((preempt_jobs_reply = send_preempt_jobs(pbs_sd, preempt_jobs_list)) == NULL) {
```

Comment:
We should also add a new type of sort to make sure that list of running jobs list has local jobs in the beginning of the list and jobs belonging to other servers behind. We already sort jobs on the basis of min_time_since_start. Maybe we will have to do a multi-sort of thing where you first sort jobs on the basis of server and when the server is same sort on their start time.

Reply:
so that we preempt jobs from the local server first? Ya I think that will be useful, I hadn't thought about that, thanks for mentioning it. Is it ok if we take that up separately?

Reply:
Fine with me, Can you please log a ticket?